### DISCUSSION TITLE
Open Source PDF editor is becoming more important than ever

### DISCUSSION THREAD

**Discussion by jaggeri** [JakubMelka/PDF4QT#51] [General]
All my bills and government letters are now delivered in PDF as compulsory - no longer in paper, and the general community needs a fair deal with a good quality open source PDF editor, more than ever before. Monthly fees for the privilege of a basic human right to PDF editing, and the planned obsolescence of "permanent" paid software is sneaky and becoming slightly immoral. It's like paying for your letter box, and a few years later the the local post office breaks it, then charges you $21.99 a month so you can physically get letters that you can stamp, write on, copy, send back to the originator and rearrange your cabinet the way you like. .... just my thoughts : (

**Comment by JakubMelka** [comment]
Yes, I agree, that is why I am programming PDF editor - to offer all people ability to edit the PDF without need to pay someone.
